test/outside-click.js

```diff
@@ -11,7 +11,8 @@ QUnit.test('overlay click', (assert) => {
   const done = assert.async()
 
   swal({
-    title: 'Overlay click'
+    title: 'Overlay click',
+    animation: false
```

Comment:
thanks for speeding up tests :+1:

Reply:
No problem! It didn't speed them up nearly as much as I estimated in #821 lol but they now run in about half the time, on my system, once the test harness is loaded.